CefSharp.Core/Internals/Messaging/Messages.h

```diff
@@ -24,6 +24,12 @@ namespace CefSharp
             const CefString kJavascriptCallbackDestroyRequest = "JavascriptCallbackDestroyRequest";
             //Message containing the result of a given js function call
             const CefString kJavascriptCallbackResponse = "JavascriptCallbackDoneResponse";
+            //Message containing a js root object for async bindings
+            const CefString kJavascriptRootObjectRequest = "JavascriptRootObjectRequest";
```

Comment:
Should have async in the name.

Likely we'll port standard JSB over to using native IPC.